cmd/site-replication.go

```diff
@@ -1430,7 +1430,9 @@ func (c *SiteReplicationSys) PeerSTSAccHandler(ctx context.Context, stsCred *mad
 	}
 
 	// Set these credentials to IAM.
-	if _, err := globalIAMSys.SetTempUser(ctx, cred.AccessKey, cred, stsCred.ParentPolicyMapping); err != nil {
+	//
+	// FIXME(aditya): Currently EntityInfo is not replicated.
```

Comment:
isn't this a problem? @donatello

Reply:
This PR is only exposing `EntityInfo` information for internal IDP and LDAP user listing and the info for this is replicated.

For LDAP, OpenID, MTLS, CustomToken based STS creds, the info is not yet replicated, but the info is not yet exposed in admin APIs - it will be exposed by updating list-access-keys (currently list-service-accounts) APIs to include STS accounts. The replication of this info is not implemented in this PR, because we need to update SR to replicate the `UserIdentity` type rather than the `Credentials` type and that is a more involved change.

The only user visible impact of not replicating this info is that when the STS account info is exposed in admin APIs, some STS accounts would have the info in Site A (where the cred is created) but not in Site B (a replication destination). However, I will fix the replication bits before we expose the STS account info, so we will have the behavior that (after that version is released), any STS accounts created post updating to that release, will have `EntityInfo` replicated.

If we really cant have this temporary inconsistency in replication, I can update this PR to not save `EntityInfo` for STS accounts in this change itself.

Reply:
This https://github.com/minio/minio/pull/17589 is towards fixing site replication so we can replicate STS better with the EntityInfo (in a subsequent change)

Reply:
We should take https://github.com/minio/minio/pull/17589 first - I will rework this change.